client/src/components/groups/group/containers/container.card.component.jsx

```diff
@@ -8,9 +8,19 @@ import './container.card.component.scss';
 
 // ContainerCard Component
 class ContainerCard extends React.Component {
+
+  // Get the daemon where container is deployed
+  getDaemon() {
+    const container = this.props.container;
+    const daemons = this.props.daemons || { items:[] };
+    const daemon = daemons.items[container.daemonId] || { 'name': 'Unknown' };
+
+    return daemon;
+  }
+
   render() {
-    const { container, daemons } = this.props;
-    const daemon = daemons.find(daemon => container.daemonId === daemon.value) || { 'name': 'Unknown' };
+    const container = this.props.container;
+    const daemon = this.getDaemon();
```

Comment:
Pas vraiment. Mais tu peux déclarer une fonction sans l'exporter. Mais ici, la fonction est dans le composant React, donc il peut être appelé de l'extérieur. Je pourrais aussi me passer des "this" et passer des paramètres.